actionview/lib/action_view/helpers/translation_helper.rb

```diff
@@ -88,7 +88,14 @@ def translate(key, options = {})
           raise e if raise_error
 
           keys = I18n.normalize_keys(e.locale, e.key, e.options[:scope])
-          content_tag('span', keys.last.to_s.titleize, :class => 'translation_missing', :title => "translation missing: #{keys.join('.')}")
+          title = "translation missing: #{keys.join('.')}"
+
+          interpolation = options.except(:default)
+          if interpolation.present?
```

Comment:
`present?` really shouldn't be used here. Use either `if interpolations.any?` or `unless interpolations.empty?` :grin: